examples/metal-nimf-2-port-connection/main.tf

```diff
@@ -37,3 +37,11 @@ module "metal_2_port_connection" {
   zside_vlan_tag        = var.zside_vlan_tag
   zside_location        = var.zside_location
 }
+
+resource "time_sleep" "wait_dl_connection" {
+  create_duration = "2m"
+}
+
+data "equinix_metal_connection" "NIMF-test" {
+  connection_id     = equinix_metal_connection.metal-connection.id
```

Comment:
Add depends_on = [time_sleep.wait_dl_connection] line in Metal Connection data source block